tutorials/README.md

```diff
@@ -1,5 +1,16 @@
-## Cortex Certifai Tutorials 
-This directory contains the self contained tutorials which walk through various parts of Certifai. 
+# Cortex Certifai Tutorials 
+This directory contains self contained tutorials which walk through basic parts of Certifai funcitonality 
 
+### Bringing In Your Own Model to Certifai Part 1
+
+ Bringing In Your Own Model to Certifai example demonstrates how to create a scan in Certifai using your own model. In this example the model being used is Logistic Regression. We will show how to configure the model to be used in Certifai, and run a scan on it with key evaluation metrics:
+
+ * Fairness
+ * Explainability
+ * Robustness
+
+The end result will be a scan definition file which contains the meta data from our scan, which can be used for running scans in the CLI and in Part 2 of this tutorial.
```

Comment:
change to . . . that contains the metadata from your scan, which is used in part two of this tutorial."